multimedia/mpv/Portfile

```diff
@@ -455,6 +456,10 @@ variant network description {Enable networking support via youtube-dl (supports
     depends_run-append      port:youtube-dl
 }
 
+variant ytdlp conflicts {network} description {Enable networking support via yt-dlp instead of youtube-dl (supports wide variety of pages)} {
```

Comment:
Brackets around `network` is not needed. Also since you made this variant conflict `network` variant, you need to do that in `network` variant as well.